server.js

```diff
@@ -651,20 +651,59 @@ proxyForAuditing.on('error', function(err, req, res) {
   serverUtils.serverError(JSON.stringify(err), req, res);
 });
 
-var couchDbVersionCheck = function(callback) {
-  db.getCouchDbVersion(function(err, version) {
-    console.log('CouchDB Version:', version);
+const nodeVersionCheck = callback => {
+  try {
+    console.log('Node Version:', process.version);
+
+    const version = process.versions.node.match(/(\d)+\.(\d)+\.(\d)+/)[1];
```

Comment:
`process.version` is the "display" version, `process.versions.node` is the non display version. The only difference AFAICT is that one has a "v" in front of it. We should probably just use the latter.